packages/sales-api-service/src/services/exceptions/temp/index.js

```diff
@@ -0,0 +1 @@
+export * from './pocl-data-validation-error.entity.js'
```

Comment:
I'm confused why this is here - couldn't you just use './pocl-data-validation-error.entity.js' directly?

Reply:
I did that to begin with, but it didn't want to import it into the file I need it in, so I recreated the way that it is exported in the dynamics-lib and it was happy with this approach. 
This is also to be removed after all the work is completed